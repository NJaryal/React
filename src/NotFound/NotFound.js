import React from 'react';

const NotFound = () => (
  <div>
    <p>No Data Found!!</p>
  </div>
);

export default NotFound;
