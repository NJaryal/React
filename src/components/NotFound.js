import React from 'react'

const NoContent = ({ data }) => <h1>{ data }</h1>
const NotFound = () => <NoContent data={ data } />

export default NotFound;