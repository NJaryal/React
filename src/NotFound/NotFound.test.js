import React from 'react';
import { shallow } from 'enzyme';
import NotFound from './NotFound';

const renderNotFound = () => {
  const component = shallow(<NotFound />);
  return component;
};

describe('NotFound component', () => {
  let component;
  beforeEach(() => {
    component = renderNotFound();
  });

  it('should render Component', () => {
    expect(component).toMatchSnapshot();
  });
});
