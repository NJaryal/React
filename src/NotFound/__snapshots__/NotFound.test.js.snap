// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component should render Component 1`] = `
<div>
  <p
    className="notFoundContent"
  >
    No Data Found!!
  </p>
</div>
`;
